### ISSUE TITLE
Profile Images userscript bug.

### ISSUE THREAD

**Issue by JourneyOver** [iFelix18/Trakt-Userscripts#2]
Seems to be a bug in the profile images userscript, where it seems to cause the links in the profile to become unclickable. It seems to be due to the shade from
```
  NodeCreationObserver.onCreation('.is-self #cover-wrapper:not(.watching-now) .full-bg.enabled', function () {
    let parent = $(this).parent()
    parent.find('.shadow.hidden-xs').remove()
    parent.append('<div class="shade"></div>')
    setProfileImage(this)
  })
```
causing the shade to be 100% on both width and height which ends up causing it to go over the links wrapper completely in turn making things unclickable.

The links I'm talking about btw are these ones here
https://i.imgur.com/PM9X39E.png

**Comment by iFelix18** [comment]
It should be fixed with [1.0.2](https://github.com/iFelix18/Trakt-Userscripts/commit/7396c82162d4a2d882439e694a98943054a40f20)

**Comment by JourneyOver** [comment]
Yep all is good now :D
